tests/test_http.py

```diff
@@ -116,7 +116,7 @@ def request(self, *args, **kwargs):
     else:
       self.num_errors -= 1
       if self.num_errors == 1:
-        raise ssl.SSLError()
```

Comment:
I'm not sure about the change to this file. If we're not going to set up infrastructure that runs our tests without the `ssl` module available, changing the test code is moot, isn't it?